cli/lsp/language_server.rs

```diff
@@ -3596,10 +3676,6 @@ impl Inner {
     let referrer = self
       .url_map
       .normalize_url(&params.referrer.uri, LspUrlKind::File);
-    if !self.is_diagnosable(&referrer) {
-      return Ok(None);
-    }
```

Comment:
Was this removed on purpose?

Reply:
Yeah since the referrer used for caching the jsx import source is a JSON file, which isn't diagnosable. Caching is only done for specific modules intentionally so it's not an issue.